### PR TITLE
Dev version1 khara

### PR DESCRIPTION
**Pull request by KharaHL**

### CODE REVIEW

**Review by fresh-ash — APPROVED** [review]

File: src/main/java/api/entities/mentors/ActiveMentors.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package api.entities.mentors;
+
+public class ActiveMentors extends MentorInformation {
+    String avatarUrl;
```
Comment (OlekLiakh) [inline src/main/java/api/entities/mentors/ActiveMentors.java]: Access modifier?

File: src/main/java/api/entities/mentors/Group.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package api.entities.mentors;
+
+public class Group {
+    Integer id;
```
Comment (OlekLiakh) [inline src/main/java/api/entities/mentors/Group.java]: Access modifier?

File: src/main/java/api/entities/mentors/MentorInformation.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
+package api.entities.mentors;
+
+public class MentorInformation {
+    Integer id;
+    String email;
+    String firstName;
+    String lastName;
+
```
Comment (OlekLiakh) [inline src/main/java/api/entities/mentors/MentorInformation.java]: Access modifiers?

File: src/main/java/api/entities/mentors/MentorInformation.java [hunk 4]
```diff
@@ -0,0 +1,40 @@
+package api.entities.mentors;
+
+public class MentorInformation {
```
Comment (OlekLiakh) [inline src/main/java/api/entities/mentors/MentorInformation.java]: Maybe a better name for this class is Mentor?

**Review by OlekLiakh — APPROVED** [review]

**Pull request merged**